### PR TITLE
docs: Update dependency graph

### PR DESCRIPTION
**Pull request by nashqueue**
## Overview

This PR aims to be a starting point for a dependency graph update

The update is adding:
- more IBC work
- connecting Sequencing API and adding Execution 
- updates to overall progress

### CODE REVIEW

File: specs/src/specs/dependency-graph.drawio.svg [hunk 1]
```diff
@@ -1,97 +1,97 @@
...
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1350px; margin-left: 381px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1250px; margin-left: 381px;">
                         <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
                                 Header Interface
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/574">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/574">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:15]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 2]
```diff
@@ -1,97 +1,97 @@
...
         <g transform="translate(-0.5 -0.5)">
             <switch>
                 <foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;">
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1140px; margin-left: 381px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1040px; margin-left: 381px;">
                         <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
-                                New Rollkit Header
+                                New Rollmint Header
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:74]: Should be Rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 3]
```diff
@@ -1,97 +1,97 @@
...
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/579">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/579">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:76]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 4]
```diff
@@ -101,913 +101,885 @@
...
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1040px; margin-left: 531px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 940px; margin-left: 531px;">
                         <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
                                 <font color="#000000">
                                     Celestia Blob Inclusion Proofs
                                 </font>
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/594">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/594">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:128]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 5]
```diff
@@ -101,913 +101,885 @@
...
         <g transform="translate(-0.5 -0.5)">
             <switch>
                 <foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;">
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 880px; margin-left: 171px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 780px; margin-left: 171px;">
                         <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
-                                P2P Rollkit Headersync
+                                P2P Rollmint Headersync
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:151]: Should be Rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 6]
```diff
@@ -101,913 +101,885 @@
...
                             </div>
                         </div>
                     </div>
                 </foreignObject>
-                <text x="230" y="884" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">
-                    P2P Rollkit Headers...
+                <text x="230" y="784" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                    P2P Rollmint Headers...
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:157]: Should be Rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 7]
```diff
@@ -101,913 +101,885 @@
...
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 630px; margin-left: 841px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 530px; margin-left: 841px;">
                         <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
-                                Rollkit Light Clients
+                                Offchain Rollkit Light Clients
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/362">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/362">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:294]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 8]
```diff
@@ -101,913 +101,885 @@
...
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1350px; margin-left: 1551px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1250px; margin-left: 1551px;">
                         <div data-drawio-colors="color: #000000; " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
                                 State Tracing
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/514">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/514">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:336]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 9]
```diff
@@ -101,913 +101,885 @@
...
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1240px; margin-left: 1611px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 1140px; margin-left: 1611px;">
                         <div data-drawio-colors="color: #000000; " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
                                 State Execution Fraud Proof generation
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/514">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/514">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:361]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 10]
```diff
@@ -101,913 +101,885 @@
...
-                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 510px; margin-left: 1471px;">
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 410px; margin-left: 1471px;">
                         <div data-drawio-colors="color: rgb(0, 0, 0); " style="box-sizing: border-box; font-size: 0px; text-align: center;">
                             <div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; pointer-events: all; white-space: normal; overflow-wrap: normal;">
                                 Make Light client lighter
                                 <br/>
-                                <a href="https://github.com/rollkit/rollkit/issues/571">
+                                <a href="https://github.com/celestiaorg/rollmint/issues/571">
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:846]: Should be rollkit/rollkit.

File: specs/src/specs/dependency-graph.drawio.svg [hunk 11]
```diff
@@ -1,97 +1,97 @@
...
                                     579
                                 </a>
                             </div>
                         </div>
                     </div>
                 </foreignObject>
-                <text x="440" y="1144" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">
-                    New Rollkit Header...
+                <text x="440" y="1044" fill="rgb(0, 0, 0)" font-family="Helvetica" font-size="12px" text-anchor="middle">
+                    New Rollmint Header...
```
Comment (tzdybal) [inline specs/src/specs/dependency-graph.drawio.svg:84]: Should be Rollkit.

**Review by tzdybal — DISMISSED** [review]
Looks like you worked with not-the-latest version of the file while updating.

**Review by tzdybal — COMMENTED** [review]
`avail-da` points to `credible neutrality` and at the same time `credible neutrality` points to `Other DA implementations`. I think 'other DA implementations` might be removed from the graph.

**Review by tzdybal — COMMENTED** [review]
Maybe you could introduce a color for "outcomes" that we're enabling, but not necessarily intend to build (like explorers, prover market, execution market, sequencer API implementations, etc)

**Review by MSevey — CHANGES_REQUESTED** [review]
pending Tomasz comments

**Review by Manav-Aggarwal — DISMISSED** [review]

**Review by tuxcanfly — APPROVED** [review]

**Review by tzdybal — CHANGES_REQUESTED** [review]
One last `P2P Rollmint Headersync` to fix.

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by tzdybal — APPROVED** [review]
:+1:

**Pull request merged**